infra/aws_bootstrap/cloudwatch.tf

```diff
@@ -0,0 +1,18 @@
+# cloudwatch configs go here
+# we need insight into the ecs tasks
+
+# ----------------------------------------------
+# Mock API
+# ----------------------------------------------
+resource "aws_cloudwatch_log_group" "mock_api" {
```

Comment:
Are all of the environments going to share the same log group? I would expect this to be created in the application templates, once per environment.